lib/rubocop/cop/rspec/context_wording.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # `context` block descriptions should start with 'when', or 'with'.
+      #
+      # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
+      # @see http://www.betterspecs.org/#contexts
+      #
+      # @example `Prefixes` configuration option, defaults: 'when', and 'with'
+      #   Prefixes:
+      #     - when
+      #     - with
+      #     - without
+      #
+      # @example
+      #   # bad
+      #   context 'the display name not present' do
+      #     ...
+      #   end
+      #
+      #   # good
+      #   context 'when the display name is not present' do
+      #     ...
+      #   end
+      class ContextWording < Cop
+        MSG = 'Start context description with %s.'.freeze
+
+        def_node_matcher :context_wording, <<-PATTERN
+          (block (send _ { :context :shared_context } $(str #bad_prefix?)) ...)
+        PATTERN
+
+        def on_block(node)
+          context_wording(node) do |context|
+            add_offense(context, message: message)
+          end
+        end
+
+        private
+
+        def bad_prefix?(description)
+          !prefixes.include?(description.partition(' ').first)
+        end
+
+        def prefixes
+          cop_config['Prefixes'] || []
+        end
+
+        def message
+          format(MSG, sentence)
+        end
+
+        def sentence
+          quoted = prefixes.map { |prefix| "'#{prefix}'" }
+          return quoted if quoted.length == 1
```

Comment:
Would be neat if this used `#size`, or `#one?` since the method operates on both collections and strings. I found myself thinking: "whenever would we get a string of length 1?" 🙂